GuessingGame.html

```diff
@@ -1,22 +1,29 @@
 <script>
+  alert("You are sitting in a bakery and smell freshly baked cupcakes. Before you can order one the cashier makes a deal with you. Pick a number between 1-100, you have 7 guesses. If you guess right you get a cupcake on the house!");
 
-    alert("You are sitting in a bakery and smell freshly baked cupcakes. 
-  Before you can order one the cashier makes a deal with you. 
-  Pick a number between 1-100, you have 7 guesses. If you guess 
-  right you get a cupcake on the house!");
+  var answer = 42,
```

Comment:
a way to take this further might be to select a different answer each time the game is played. the if below would have to change, then.